crucible-llvm-cli/test-data/ptr.cbl

```diff
@@ -21,4 +21,12 @@
     (assert! (equal? vblk0 vblk) "stored block numbers equal")
     (assert! (equal? voff0 voff) "stored offsets equal")
 
+    (let g (resolve-global "malloc"))
+    (let gblk (ptr-block 64 g))
+    (assert! (not (equal? gblk 0)) "malloc block number nonzero")
+    (let h (load-handle (Ptr 64) ((Ptr 64)) g))
```

Comment:
Agreed, that was an oversight on my part!

> Speaking of which, the type signature of malloc is i8* malloc(size_t). Does this mean that size_t becomes (Ptr 64) here? I think so, but it would be worth saying as much somewhere.

Yep! These types are at the Crucible-LLVM level, so there is no such thing as `size_t` or `i8*`, just `LLVMPointerType` (written `Ptr`). Will describe that in the README.